pydarn/plotting/maps.py

```diff
@@ -1030,3 +1030,117 @@ def plot_potential_contours(cls, fit_coefficient: list, lat_min: list,
                     color=pot_minmax_color, zorder=5.0)
         plt.scatter(np.radians(min_mlon), min_mlat, marker='_', s=70,
                     color=pot_minmax_color, zorder=5.0)
+        
+    
+
+    class TimeSeriesParams(Enum):
+        """
+        Enum class to hold the parameters that can be plotted
+        Members
+        ------------
+        NUM_VECTORS:
+        the number of aviliable vectors in the map
+        IMF_BY:
+        The magnitude of the IMF By component in nT
+        IMF_BZ:
+        The magnitude of the IMF Bz component in nT
+        KP:
+        The Kp index
+        HMB_MAX:
+        The farthest equatorward extent of the HMB
+
+        """
+        NUM_VECTORS = 'num_vectors'
+        IMF_BY = 'imf_by'
+        IMF_BZ = 'imf_bz'
+        KP = 'kp'
+        HMB_MAX = 'hmb_max'     
+        
+    @classmethod
+    def plot_time_series(cls, dmap_data: List[dict],
+                     parameter: Enum = TimeSeriesParams.NUM_VECTORS,
+                     start_record: int = 0,end_record: int = 1, start_time: dt.datetime = None,
+                     end_time: dt.datetime = None):
```

Comment:
```suggestion
                     end_time: dt.datetime = None, **kwargs):
```
Add **kwargs for matplotlib options in plotting. 
Also add **kwargs to each of the plot.plot calls below to get them into the plots. 
This will allow the user to be able to choose plotting options, like using `color='r'` in the method call will find it's way through to the plot call and you can plot with a red line instead. 
If you want any help with this let me know, this is sometimes a hard one to get right.

Reply:
I'll get working on these changes and hopefully have an updated version out soon 👍 .